syntax = "proto2";

package opentxs.proto;
option java_package = "org.opentransactions.proto";
option java_outer_classname = "OTPaymentCode";
option optimize_for = LITE_RUNTIME;

import public "Enums.proto";

message PaymentCode {
    optional uint32 version = 1;
    optional bytes key = 2;
    optional bytes chaincode = 3;
    optional bool bitmessage = 15;
    optional uint32 bitmessageVersion = 16;
    optional uint32 bitmessageStream = 17;
}
